src/views/wallet/WalletSettingsScreen.tsx

```diff
@@ -0,0 +1,162 @@
+    const keystore: string = window.localStorage.getItem('binance') || ''
+    return 'data:text/plain;charset=utf-8,' + encodeURIComponent(keystore)
+  }, [])
+
+  const fileName = (): string => userAccount.address.concat('-keystore.txt')
+
+  const lockWallet = useCallback(async () => {
+    console.log('locking wallet...')
+  }, [])
+  const removeWallet = useCallback(async () => {
+    console.log('removing wallet...')
+  }, [])
+
+  return (
+    <Row>
+      <Title level={3}>Settings</Title>
+      <Col span={24}>
+        <Row gutter={[16, 16]}>
+          <Col md={{ span: 12 }} lg={{ span: 12 }} xl={{ span: 12 }}>
+            <Paragraph strong>Wallet Management</Paragraph>
+            <Card>
+              <Row>
+                <Col span={12}>
+                  <Card bordered={false}>
+                    <Button type="ghost" shape="round" block href={downloadLink} download={fileName()}>
+                      Export Keystore
+                    </Button>
+                  </Card>
+                </Col>
+                <Col span={12}>
+                  <Card bordered={false}>
+                    <Button type="ghost" shape="round" block onClick={lockWallet}>
+                      Lock Wallet
+                    </Button>
+                  </Card>
+                </Col>
+                <Col span={12}>
+                  <Card bordered={false}>
+                    <Button type="ghost" shape="round" block disabled={true}>
+                      View Phrase
+                    </Button>
+                  </Card>
+                </Col>
+                <Col span={12}>
+                  <Card bordered={false}>
+                    <Button type="ghost" danger shape="round" block onClick={removeWallet}>
+                      Remove Wallet
+                    </Button>
+                  </Card>
+                </Col>
+              </Row>
+            </Card>
+            <Paragraph strong>Network Client</Paragraph>
+            <Card>
+              <Row>
+                <Col md={{ span: 24 }} lg={{ span: 12 }}>
+                  <Text strong>Account:</Text>
+                  <Paragraph ellipsis>{userAccount.address}</Paragraph>
+                </Col>
+                <Col md={{ span: 24 }} lg={{ span: 12 }}>
+                  <Text strong>Keystore Version:</Text>
+                  <Paragraph>{userAccount.keystore.version}</Paragraph>
+                </Col>
+                <Col md={{ span: 24 }} lg={{ span: 12 }}>
+                  <Text strong>Type:</Text>
+                  <Paragraph>{client.network}</Paragraph>
+                </Col>
+                <Col md={{ span: 24 }} lg={{ span: 12 }}>
+                  <Text strong>Chain ID:</Text>
+                  <Paragraph>{client.chainId}</Paragraph>
+                </Col>
+              </Row>
+            </Card>
+          </Col>
+
+          <Col sm={{ span: 24 }} md={{ span: 12 }} lg={{ span: 12 }} xl={{ span: 12 }}>
```

Comment:
[nitpick] Same as above: `md` should work (no need to use same values for `lg`, `xl`)